samples/bootloader/src/main.c

```diff
@@ -26,9 +33,23 @@ static void validate_and_boot(const struct fw_info *fw_info)
 		return;
 	}
 
+	printk("Firmware version %d\r\n", fw_info->version);
+
+	if (fw_info->version > get_monotonic_version(NULL)) {
```

Comment:
@hakonfam Do you have an opinion on whether this should be in bl_boot instead?

Reply:
It will not be used by botnet, so it must be configurable (which I guess it already is), and keeping stuff away from main is always good, right? I would propose that you mock it up, and go with whatever looks best.

Reply:
Ok, in that case I think it makes most sense where it is. I might move the prints into the function in bl_validation